src/source_sampling.h

```diff
@@ -88,6 +90,35 @@ namespace pyne {
     std::vector<double> prob; /// Probabilities.
     std::vector<int> alias; /// Alias probabilities.
   };
+
+  // class Source particle
+  class SourceParticle {
+    public:
+    SourceParticle();
+    /// Constructor for source particle
+    /// \param x The x coordinate of the source particle
+    /// \param y The y coordinate of the source particle
+    /// \param z The z coordinate of the source particle
+    /// \param e The energy of the source particle
+    /// \param w The weight of the source particle
+    /// \param c The cell number of the source particle
+    SourceParticle(double x,
+                   double y,
+                   double z,
+                   double e,
+                   double w,
+                   int c);
+    ~SourceParticle();
+
+    double x; // x coordinate
```

Comment:
If you are providing access functions for these, perhaps they can/should be private member?